public/data/KendallDoesCoding.json

```
@@ -9,6 +9,11 @@
       "url": "https://kendalldoescoding.gq/",
       "icon": "globe"
     },
+    {
+      "name": "A Very Mogul Christmas (Currently Working On)",
+      "url": "https://kendalldoescoding.gq/mogulchristmas",
+      "icon": "globe"
+  },
```

```suggestion
    {
      "name": "A Very Mogul Christmas (Currently Working On)",
      "url": "https://kendalldoescoding.gq/mogulchristmas",
      "icon": "globe"
    },
```
Add two spaces before `},`. To follow the format of the JSON file.

---

Hey, @vikasganiga05 we don't have to worry about the indentation and formatting of the file, we have set up the Actions for that :)

---

Hey, @vikasganiga05 we don't have to worry about the indentation and formatting of the file, we have set up the Actions for that :)
Hope I'm right @schmelto?

---

Oh! I forgot that Thanks!
But the github-action has not formated the file. When it will be formatted? I am new to github-actions.

---

@Pradumnasaraf Oh,  I didn't know that.

@vikasganiga05 Thanks, will commit this though.

---

Welcome! :slightly_smiling_face: